serverless/stacks/AppStack.ts

```diff
@@ -22,15 +27,34 @@ export function AppStack({ stack }: StackContext) {
       onnxUri: "string",
       confusionMatrixUri: "string",
       aucRocUri: "string"
-    },
-    primaryIndex: {
-      partitionKey: "runId",
-    },
-    globalIndexes: {
-      "TrainspaceIndex": { partitionKey: "trainspaceId", sortKey: "timestamp"},
-      "UserIndex": {partitionKey: "userId"}
-    },
+    };
+
+    table_attributes_exists(exists, tableFieldBody).then(tableAttributesExists => {
+      if (!tableAttributesExists) {
+        new Table(stack, "trainspace-run", {
+          fields: {
+            runId: "string",
+            trainspaceId: "string",
+            userId: "string",
+            timestamp: "string",
+            resultCsvUri: "string",
+            modelPtUri: "string",
+            onnxUri: "string",
+            confusionMatrixUri: "string",
+            aucRocUri: "string"
+          },
+          primaryIndex: {
+            partitionKey: "runId",
+          },
+          globalIndexes: {
+            "TrainspaceIndex": { partitionKey: "trainspaceId", sortKey: "timestamp"},
+            "UserIndex": {partitionKey: "userId"}
+          },
```

Comment:
What if I want to create a new GSI or User index. Can we update the PR to be resilient for those cases? @alantao912

Reply:
I don't think the current approach is correct. I'm looking into other solutions.